utils/inspect-response.js

```diff
@@ -1,6 +1,31 @@
 const explorer = require('./explorer');
-
+const config = require('../get-config')();
+const chalk = require('chalk');  // colorize output
 const util = require('util');
+
+
+const checkExistAccount = (error, options) => {
+    if(!String(error).includes('does not exist while viewing')) return false;
+
+    console.log(chalk`\n{bold.red Account {bold.white ${options.accountId}} is not found in {bold.white ${config.helperAccount}} network.\n}`);
+
+    const suffix = String(options.accountId).match(/[^\.]*$/gi)[0];
+            
+    switch(suffix) {
+        case 'near':
+            console.log(chalk`{bold.white Use export NEAR_ENV=mainnet to use MainNet accounts. \n}`);
+            break;
+        case 'testnet': 
+            console.log(chalk`{bold.white Use export NEAR_ENV=testnet to use TestNet accounts. \n}`);
+            break;
+        case 'betanet': 
+            console.log(chalk`{bold.white Use export NEAR_ENV=betanet to use BetaNet accounts. \n}`);
+            break;
+    }
+
+    return true;
+}
```

Comment:
another nit-pick: missing semicolon. Please, check output from Travis CI, there is a banch of other minor issues:
`12:55  error  Unnecessary escape character: \.               no-useless-escape
  15:1   error  Expected indentation of 4 spaces but found 8   indent
  16:1   error  Expected indentation of 8 spaces but found 12  indent
  17:1   error  Expected indentation of 8 spaces but found 12  indent
  18:1   error  Expected indentation of 4 spaces but found 8   indent
  19:1   error  Expected indentation of 8 spaces but found 12  indent
  20:1   error  Expected indentation of 8 spaces but found 12  indent
  21:1   error  Expected indentation of 4 spaces but found 8   indent
  22:1   error  Expected indentation of 8 spaces but found 12  indent
  23:1   error  Expected indentation of 8 spaces but found 12  indent
  27:2   error  Missing semicolon                              semi`